eZ/Publish/Core/Persistence/Legacy/Content/UrlWildcard/Gateway/DoctrineDatabase.php

```diff
@@ -186,6 +237,35 @@ public function countAll(): int
         return (int) $query->execute()->fetchColumn();
     }
 
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotImplementedException
+     */
+    protected function doCount(Criterion $criterion): int
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->select('COUNT(DISTINCT url_wildcard.id)')
```

Comment:
`ezurlwildcard.id` is a primary key. What is the reason for this `DISTINCT`?
Anyway:
```suggestion
            ->select($this->connection->getDatabasePlatform()->getCountExpression('url_wildcard.id'))
```